packages/system/capi-providers/templates/providers.yaml

```diff
@@ -13,7 +13,7 @@ spec:
   deployment:
     containers:
     - name: manager
-      imageUrl: ghcr.io/kvaps/test:cluster-api-control-plane-provider-kamaji-v0.6.0-fix7
+      imageUrl: ghcr.io/kvaps/test:cluster-api-control-plane-provider-kamaji-v0.7.1-fix
```

Comment:
This can be removed after v0.8.0 release

https://github.com/clastix/cluster-api-control-plane-provider-kamaji/milestone/13